CONTRIBUTING.md

```diff
@@ -38,6 +38,8 @@ Since the goal is to have a hundred municipalities into the system, it is reason
 
 Won't be perfect or as generic as we would like but maybe enough to populate a database to train Machine Learning and Natural Language Processing solutions. Meaning: the code you will find - and write - in the [`processing/gazette`](processing/gazette) folder is under heavy experimentation. Doesn't need to be beautiful or scalable if it works for specific municipalities.
 
+We are interested only in the bidding exemptions (dispensas de licitação) and bidding ineligibilities (inexigibilidade de licitação). 
```

Comment:
It is great, to be more precise, I would add "**For now**, we are (...)".

Reply:
Good point. I just added your suggestion.